Sources/Testing/Support/FileHandle.swift

```diff
@@ -0,0 +1,224 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+internal import TestingInternals
+
+#if !SWT_NO_FILE_IO
```

Comment:
A newline after this to match the `#endif` would be nice.

Reply:
Fixed it by deleting the one at the end instead.